src/videojs.markers.js

```diff
@@ -231,21 +227,26 @@ type Marker = {
       ) {
         if (overlayIndex !== currentMarkerIndex) {
           overlayIndex = currentMarkerIndex;
-          breakOverlay && breakOverlay.find('.vjs-break-overlay-text').html(setting.breakOverlay.text(marker));
+          if (breakOverlay) breakOverlay.querySelector('.vjs-break-overlay-text').innerHTML = setting.breakOverlay.text(marker);
         }
 
-        breakOverlay && breakOverlay.css('visibility', "visible");
+        if (breakOverlay) breakOverlay.style.visibility = "visible";
       } else {
         overlayIndex = NULL_INDEX;
-        breakOverlay && breakOverlay.css("visibility", "hidden");
+        if (breakOverlay) breakOverlay.style.visibility = "hidden";
```

Comment:
The && style does not work when right hand side has an assignment.

I can switch to multi line if statement with braces.